src/update-feeds-loops/pending-transaction-info.ts

```diff
@@ -40,22 +40,29 @@ export const updatePendingTransactionsInfo = (
   } = getState();
 
   const currentTimestamp = Math.floor(Date.now() / 1000);
-  for (const feed of batch) {
-    const { dapiName, dataFeedId, decodedDapiName, updateParameters, dataFeedTimestamp } = feed;
-
-    const isFeedUpdatable = feedsToUpdate.some(
+  for (const { beaconsWithData, dapiName, dataFeedId, decodedDapiName, updateParameters, dataFeedTimestamp } of batch) {
+    const updatableFeed = feedsToUpdate.find(
       (updatableFeed) =>
         updatableFeed.dataFeedInfo.dapiName === dapiName && updatableFeed.dataFeedInfo.dataFeedId === dataFeedId
     );
 
+    if (updatableFeed && !updatableFeed.shouldUpdateBeaconSet && beaconsWithData.length > 1) {
+      // Feed is a beacon set and it will only update its beacons. Therefore no need to set nor clear pending transaction info.
+      logger.warn(
+        'Data feed is a beacon set that does not require an update but some of its beacons do. Skipping pending transaction info update.',
+        { dapiName: decodedDapiName, dataFeedId }
```

Comment:
nit: I think dapiName and dataFeedId may already be logged because of the log context propagation. Not sure about this though.

Reply:
They are also added to logger context on lines 77 and 92 so I'll keep them here